benchmark-service/benchmark/benchmark.py

```diff
@@ -183,8 +191,33 @@ async def show_name(request: web.Request, userdata) -> web.Response:  # pylint:
 
 @router.get('/')
 @router.get('')
+async def index(request):
+    userdata = {}
+    global benchmark_data
+    d = {
+        'dates': benchmark_data['dates'],
+        'geo_means': benchmark_data['geo_means'],
+        'pr_ids': benchmark_data['pr_ids']
+    }
+    assert len(d['dates']) == len(d['geo_means']), d
+    df = pd.DataFrame(d)
+    if not df.dates.empty:
+        fig = px.line(df, x=df.dates, y=df.geo_means, hover_data=['commits'])
```

Comment:
The commits here doesn't seem right to me. Are you sure the hover data specification is correct?